### ISSUE TITLE
Node ObjectLiteralExpression did not pass test 'isParameterDeclaration'.

### ISSUE THREAD

**Issue by carlo-quinonez** [ptc-iot-sharing/ThingworxVSCodeProject#32]
1. Started with a fresh clone of this repo.
2. ran `npm i` (node 14.17)
3. ran `gulp` then exited command
4. ran `npm run build`

I get the following error:

![image](https://user-images.githubusercontent.com/53790047/128606097-225a2eb8-e8ce-427b-91b2-2caaac0c413f.png)

**Comment by BogdanMihaiciuc** [comment]
I'm not able to reproduce this locally. Do you get this building an unmodified clone of the repo?

I'm running node v15, but I'm pretty sure it's not relevant for this error. 

Are you using a version of typescript different from the one with which the transformer was built on (4.3.4)? This causes issues because the constant values for node kinds seem to change sometimes between versions.

**Comment by carlo-quinonez** [comment]
Yes, I made a fresh clone of the repo - no changes what so ever. I just tried again with Node 15 and made sure to use version 4.3.4 of typescript, but I got the same error.

**Comment by BogdanMihaiciuc** [comment]
Can you run a debug build (it should be set up as a configuration is vscode already) and set a breakpoint in the `isParameterDeclaration` function in `node_modules/typescript/lib/typescript.js` and let me know the value of `root.kind` where it fails?

If you can use the call stack to let me know where in the parsing it fails this might also help.
